VirtusLab-Open-Source/strapi-plugin-navigation
# feat: populate internal path with content type field [CU-2jtv8xj]

## Ticket

#152 

## Summary

You can connect the path to attributes of a related content type. When configured path (if empty) will be populated with the attribute from the selected related. Many attributes can be connected to the path. It will fall back to the next one if the previous field was empty in the related entity. 
 
## Test Plan

- Create project 
- Configure path default param on the Settings page 
- Try to add a new element with the configured content type
- The empty internal path should be populated with fields from the selected content type entity
